src/Compiler/CMakeLists.txt

```diff
@@ -100,7 +100,7 @@ endforeach()
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/../../bin)
 
 set(CMAKE_CXX_FLAGS_DEBUG
-"${CMAKE_CXX_FLAGS_DEBUG} ${TCMALLOC_COMPILE_OPTIONS} -Werror -Wall -O0 -g ${MSYS_COMPILE_OPTIONS} ${MY_CXX_WARNING_FLAGS} ${MEM_SANITIZER_FLAGS}")
+"${CMAKE_CXX_FLAGS_DEBUG} ${TCMALLOC_COMPILE_OPTIONS} -Wall -O0 -g ${MSYS_COMPILE_OPTIONS} ${MY_CXX_WARNING_FLAGS} ${MEM_SANITIZER_FLAGS}")
```

Comment:
We want as many subsystems as possible to treat warnings as errors. So this is not a good change.

Reply:
Only when we did not write a subsystem and have little control on the code would we accept warnings.

Reply:
This flag affect gcc only. msvc failed on it. I moved it into the correct place.